source/Xunit.Gherkin.Quick/ParameterHelper.cs

```diff
@@ -0,0 +1,44 @@
+using Gherkin.Ast;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Xunit.Gherkin.Quick
+{
+    internal static class ParameterHelper
+    {
+        public static List<object> GetParamValues(MethodInfo method, Step step, List<string> specificationParameters)
+        {
+            var methodParams = method.GetParameters();
+            var dataTable = (step.Argument is DataTable) ? (DataTable)step.Argument : null;
```

Comment:
This can be slightly optimized, if you cast just once (`is` operator does cast, as well as you have an explicit cast).
Instead, you can cast just once:
`var dataTable = step.Argument as DataTable;`

Reply:
Good call, fixed